documentation/sphinx/source/rangelock.rst

```diff
@@ -58,7 +61,7 @@ If the execution is failed, no range is locked/unlocked.
 
 Get exclusive read locks on the input range
 
-``ACTOR Future<std::vector<KeyRange>> getExclusiveReadLockOnRange(Database cx, KeyRange range);``
+``ACTOR Future<std::vector<std::pair<KeyRange, RangeLockState>>> findExclusiveReadLockOnRange(Database cx, KeyRange range);``
```

Comment:
Shouldn't the method be called get ExclusiveReadLock?  Not findExclusiveReadLock.  (OnRange is not needed?)

Reply:
I think `getExclusiveReadLockOnRange` is misleading. People may think getExclusiveReadLockOnRange as grabbing the lock.